src/plugins/CustomAppIcons/index.tsx

```diff
@@ -68,9 +68,7 @@ export default definePlugin({
 
 
     start() {
-        console.log("Well hello there!, CustomAppIcons has started :)");
         console.warn("Warning: CustomAppIcons is in beta");
```

Comment:
Can you fix this too? Regular `console` is not working in Vencord. You need to use `Logger` instead.

Reply:
Ok. Let me check source code real quick